src/components/MultiHopTrade/components/FiatRamps/hooks/useGetRampQuotes.tsx

```
@@ -0,0 +1,86 @@
+    return ramps?.byAssetId[assetId]?.[direction].map(fiatRampId => supportedFiatRamps[fiatRampId])
+  }, [ramps, assetId, direction])
+
+  const rampQuoteQueries = useQueries({
+    queries:
+      supportedRamps?.map(fiatRamp => ({
+        queryKey: [...queryKey, fiatRamp.id],
+        queryFn: () => {
+          switch (fiatRamp.id) {
+            case 'OnRamper':
+              if (!onramperCurrencies) throw new Error('Onramper currencies not found')
+              const crypto = findOnramperTokenIdByAssetId(assetId, onramperCurrencies)
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)